crates/wash-cli/tests/wash_start.rs

```diff
@@ -70,5 +95,29 @@ async fn integration_start_provider_serial() -> Result<()> {
         serde_json::from_slice(&output.stdout).context("failed to parse output")?;
     assert!(cmd_output.success, "command returned success");
 
+    // Test stopping using only aliases, yes I know this mixes stop and start, but saves on copied
+    // code
+    let status = Command::new(env!("CARGO_BIN_EXE_wash"))
+        .args([
+            "stop",
+            "provider",
+            "server",
+            "wasmcloud:httpserver",
+            "--output",
+            "json",
+            "--timeout-ms",
+            "40000",
+            "--ctl-port",
+            &wash_instance.nats_port.to_string(),
+        ])
+        .kill_on_drop(true)
+        .stdout(Stdio::null())
+        .stderr(Stdio::null())
+        .status()
+        .await
+        .context("failed to start actor")?;
```

Comment:
Would you mind hanging the provider stop also off of `impl TestWashInstance` ? Definitely something people would want to do commonly from tests

Reply:
Also don't want to add this as part of the fixture stuff because it is explicitly testing that I can stop it, not being used as some way of cleaning it up

Reply:
See #1032